tests/config.ini

```diff
@@ -0,0 +1,5 @@
+[agent_settings]
+SCALYR_API_KEY=0cFzhZxkn4HVSOsXkb0Zsj_N8IRq1j_26NqllaXzn0xA-
```

Comment:
It would be better to provide those are Circle CI environment variables which are handled are secrets by default (aka are not leaked in the build logs, etc.).

I believe we also already do that for some of the end to end tests.

Reply:
Oops, I forgot to remove it:)

This path should be in `.gittignore`, and it can be created to configure local tests.
The idea was to use it only locally when there are no environment variables.

Reply:
No problem, I also assume you already rotated those credentials :)